File: crates/tools/mudu_gen/src/src_check/fixtures/bad_call.py
# Negative fixture: arity mismatch and unknown table.
from mududb.db import Database
from mududb.sql import Params, SqlStmt


def get_balance(db: Database, user_id: int) -> int:
    rows = db.query(
        SqlStmt("SELECT balance FROM wallets WHERE user_id = ?"),
        Params().bind(0, user_id).bind(1, 0),
    )
    return 0


def delete_wallet(db: Database, user_id: int) -> int:
    return db.command(
        SqlStmt("DELETE FROM wallet WHERE user_id = ?"),
        Params().bind(0, user_id),
    )

File: crates/tools/mudu_gen/src/src_check/fixtures/bad_call.go
package main

// Negative fixture: arity mismatch and unknown table.
func getBalance(session muduOid, userID int64) (int64, error) {
	rows, err := sysQuery(session, "SELECT balance FROM wallets WHERE user_id = ?", userID, int64(0))
	_, _ = rows, err
	return 0, nil
}

func deleteWallet(session muduOid, userID int64) (int64, error) {
	return sysCommand(session, "DELETE FROM wallet WHERE user_id = ?", userID)
}
